sqlmesh/core/model/definition.py

```diff
@@ -1289,20 +1327,23 @@ def _create_model(
     return t.cast(Model, model)
 
 
-def _extract_sql_model_select(
+SEED_INSERT_MACRO_CALL = d.parse("@SEED_INSERT()")[0]
```

Comment:
@tobymao @crericha open to suggestions around naming this macro call

Reply:
what about @INSERT_SEED()